lib/Pg/Reindex.pm

```diff
@@ -0,0 +1,783 @@
+package Pg::Reindex;
+require Exporter;
+@ISA       = qw(Exporter);
```

Comment:
Bad. No module should inherit everything from `Exporter`. The only thing you usually need is `import`. So...

```perl
use Exporter qw/import/;
```